Lib/glyphsets/test_strings.json

```diff
@@ -1,35 +1,35 @@
 {
-    "GF_Latin_Kernel": {
+    "GF_Latin_Core": {
```

Comment:
By changing this key, we now won't have test strings to GF_Latin_Kernel. What you probably want to do is append the new strings you've provided to a new key called "GF_Latin_Core".

Reply:
But this test page was already for Latin Core, the default set also covered by FB check